### PR TITLE
Systematic FNT

### PR DESCRIPTION
**Pull request by lamphamsy**
Add encode/decode for systematic FNT codes

### CODE REVIEW

**Comment by vrancurel** [comment]
amazing !

File: src/fec_rs_fnt.h [hunk 1]
```diff
@@ -50,6 +50,16 @@ namespace fec {
  */
 template <typename T>
 class RsFnt : public FecCode<T> {
+  private:
+    // buffers for intermediate symbols used for systematic FNT
+    std::unique_ptr<vec::Buffers<T>> inter_words;
+    // buffers for suffix symbols of cocdewords used for systematic FNT
```
[the remark this answers is not in the document]
Comment (lamphamsy) [reply src/fec_rs_fnt.h]: Thanks, it should be `codewords` :)

File: src/fec_vectorisation.cpp [hunk 2]
```diff
@@ -50,7 +50,7 @@ void RsFnt<uint16_t>::encode_post_process(
 {
```
[the remark this answers is not in the document]
Comment (lamphamsy) [reply src/fec_vectorisation.cpp]: It will be squashed

[the remark this answers is not in the document]
Comment (lamphamsy) [reply src/fec_vectorisation.cpp]: I will rewrite the commit message :)

**Comment by lamphamsy** [comment]
I've just rebase + squash commits

**Pull request merged**